src/sentry/api/serializers/rest_framework/dashboard.py

```diff
@@ -478,6 +481,14 @@ def validate(self, data):
                     {"displayType": "displayType is required during creation."}
                 )
 
+        # Validate limit on chart widgets
```

Comment:
We should have a comment here that explains why a limit is required. I forgot if the original issue points it out but if not I can help you create one if you need